pilot/pkg/config/kube/crdclient/gen/main.go

```diff
@@ -51,6 +53,7 @@ type ConfigData struct {
 func MakeConfigData(schema collection.Schema) ConfigData {
 	out := ConfigData{
 		Namespaced:      !schema.Resource().IsClusterScoped(),
+		IstioStatus:     !strings.Contains(schema.VariableName(), "K8S"),
```

Comment:
There are k8s types we will want to handle status for as well. Do we need to filter? I think everything is standardizing on https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Condition across the ecosystem anyways?

@therealmitchconnors

Reply:
i mean if i dont filter an error happens because the k8s resources statuses are not `IstioStatus`.

for example, `TCPRoute` has `TCPRouteStatus`, not `IstioStatus`.

Reply:
How does this work now that only WorkloadEntry has status defined?

Reply:
I think you mean `WorkloadGroup`.

And that one change was just for testing stuff, I should be adding the status field to all of the Pilot Fields Now.